### DISCUSSION TITLE
Optuna in C++

### DISCUSSION THREAD

**Discussion by DeLaRiva** [optuna/optuna#3885] [Q&A] [answered]
Hi,

is it somehow possible / doable to run Optuna in C++?
Is it promising to try it with a wrapper or the like?

Thank you!

**Comment by nzw0301** [comment, marked as answer]
How about using https://optuna.readthedocs.io/en/stable/reference/cli.html? I suppose C++ can execute optuna via CLI.

**Comment by DeLaRiva** [comment]
Thank you, I'll have a close look at it!
